ember-toucan-core/src/components/form/fields/checkbox-group.hbs

```diff
@@ -1,20 +1,41 @@
-<Form::Fieldset
-  @label={{@label}}
-  @hint={{@hint}}
-  @error={{@error}}
-  @isDisabled={{@isDisabled}}
-  aria-invalid={{if @error "true"}}
-  ...attributes
->
-  {{yield
-    (hash
-      CheckboxField=(component
-        (ensure-safe-component this.CheckboxFieldComponent)
-        name=@name
-        onChange=this.handleInput
-        isDisabled=@isDisabled
-        selectedValues=@value
-      )
-    )
-  }}
-</Form::Fieldset>
\ No newline at end of file
+<Form::Field as |field|>
+  <fieldset
+    aria-describedby="{{if @error field.errorId}} {{if @hint field.hintId}}"
+    disabled={{@isDisabled}}
+    {{! attribute aria-invalid is not supported by the element fieldset with the implicit role of group }}
+    ...attributes
+  >
+    <legend
+      class="type-md-tight text-body-and-labels block"
+      data-label
+    >{{@label}} {{yield to="label"}}</legend>
```

Comment:
General question: do we really want to render both the argument and the block, in case both are provided? This is not wrong per se, but maybe confusing? We _could_ instead also give one a higher priority, and render only that:

Arg has priority:
```hbs
{{#if @label}}
  {{@label}}
{{else}}
  {{yield to="label"}}
{{/if}}
```

or block has priority:
```hbs
{{#if (has-block "label")}}
  {{yield to="label"}}
{{else}}
  {{@label}}
{{/if}}
```

What do you think?

Reply:
Good question, I think rendering one or the other makes sense.

@ynotdraw  thoughts?

Reply:
I agree on choosing one or the other but not both! I'm leaning towards block having priority for no real reason other but have 0 strong opinions.